webrtc.html

```diff
@@ -1909,24 +1909,22 @@
 
         <dl class="idl" data-merge="RTCSessionDescriptionInit" title=
         "interface RTCSessionDescription">
-          <dt>Constructor (optional RTCSessionDescriptionInit
-          descriptionInitDict)</dt>
+          <dt>Constructor (RTCSessionDescriptionInit descriptionInitDict)</dt>
 
           <dd>The <dfn id=
           "dom-sessiondescription"><code>RTCSessionDescription()</code></dfn>
-          constructor takes an optional dictionary argument,
+          constructor takes an dictionary argument,
           <var>descriptionInitDict</var>, whose content is used to initialize
-          the new <code><a>RTCSessionDescription</a></code> object. If a
-          dictionary key is not present in <var>descriptionInitDict</var>, the
-          corresponding attribute will be initialized to null. If the
-          constructor is run without the dictionary argument, all attributes
-          will be initialized to null. This class is a future extensible
-          carrier for the data contained in it and does not perform any
-          substantive processing.</dd>
+          the new <code><a>RTCSessionDescription</a></code> object. If
+          the <code>sdp</code> dictionary key is not present
```

Comment:
Dictionaries have members, not keys. I know this mistake predates this patch, but I checked and this looks like the sole place in this spec where we refer to a dictionary 'key' in this way, so maybe a good time to nip this?